src/platform_impl/windows/event_loop.rs

```diff
@@ -526,7 +559,7 @@ impl<T: 'static> Clone for EventLoopProxy<T> {
 impl<T: 'static> EventLoopProxy<T> {
     pub fn send_event(&self, event: T) -> Result<(), EventLoopClosed<T>> {
         unsafe {
-            if winuser::PostMessageW(self.target_window, *USER_EVENT_MSG_ID, 0, 0) != 0 {
+            if PostMessageW(self.target_window, *USER_EVENT_MSG_ID, 0, 0) == true.into() {
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)